src/Core/Checkout/Customer/SalesChannel/RemoveWishlistProductRoute.php

```diff
@@ -60,21 +60,31 @@ public function getDecorated(): AbstractRemoveWishlistProductRoute
      * @Since("6.3.4.0")
      * @OA\Delete(
      *      path="/customer/wishlist/delete/{productId}",
-     *      summary="Delete customer wishlist",
+     *      summary="Remove a prduct from a wishlist",
+     *      description="Removes a product from a customer's wishlist.
+
+     **Important constraints**
+
+     * Anonymous (not logged-in) customers can not have wishlists.
+     * The wishlist feature has to be activated.",
      *      operationId="deleteProductOnWishlist",
      *      tags={"Store API", "Wishlist"},
      *      @OA\Parameter(
      *        name="productId",
      *        in="path",
-     *        description="Product Id",
-     *        @OA\Schema(type="string"),
+     *        description="Product id",
```

Comment:
I would go with `ID` in parameter descriptions. It's more understandable if the description of a parameter contains a little bit more information - compare with `DeleteAddressRoute`.